### PR TITLE
feat: add component for primary button

### PR DESCRIPTION
**Pull request by FilipeR13**

### CODE REVIEW

File: lib/atomic_web/components/button.ex [hunk 1]
```diff
@@ -2,6 +2,16 @@ defmodule AtomicWeb.Components.Button do
   @moduledoc false
   use AtomicWeb, :component
 
+  attr :url, :string, required: true
+  slot :inner_block, required: true
```
Comment (ruilopesm) [inline lib/atomic_web/components/button.ex:6]:
```suggestion
  slot :inner_block, required: true

```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Run `mix format` in order to pass CI.

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
This looks awesome :star_struck: , there's only 2 buttons left:

- New Partner button at `partner_index_page`
- New Board member button at `board_index_page` 

Could you update them?
After that it's ready to go !!

**Review by MarioRodrigues10 — APPROVED** [review]
Good job !! :rocket: :orange_heart:

**Pull request merged**